src/core/dbus-unit.c

```diff
@@ -1127,6 +1127,118 @@ static int property_get_ip_counter(
         return sd_bus_message_append(reply, "t", value);
 }
 
+int bus_unit_method_attach_processes(sd_bus_message *message, void *userdata, sd_bus_error *error) {
+
+        _cleanup_(sd_bus_creds_unrefp) sd_bus_creds *creds = NULL;
+        _cleanup_(set_freep) Set *pids = NULL;
+        Unit *u = userdata;
+        const char *path;
+        int r;
+
+        assert(message);
+
+        /* This migrates the processes with the specified PIDs into the cgroup of this unit, optionally below a
+         * specified cgroup path. Obviously this only works for units that actually maintain a cgroup
+         * representation. If a process is already in the cgroup no operation is executed – in this case the specified
+         * subcgroup path has no effect! */
+
+        r = mac_selinux_unit_access_check(u, message, "start", error);
```

Comment:
IIUC, if we wanted something different than "start", if would be necessary to amend selinux policy. Might be worth it. Maybe we should reach out to selinux maintainers for comments?